src/Validators/Validator.Strings.cs

```diff
@@ -73,4 +73,22 @@ internal static void ThrowIfNotEquals(string value, string paramName, ExceptionC
             ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should be equal to '{otherString}'.");
         }
     }
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void IfLengthEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, int length)
```

Comment:
```suggestion
    internal static void ThrowIfLengthEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, int length)
```
H&B

Reply:
😅 missed that convention there. Will fix it